svc/pkg/cluster/worker/src/workers/server_install/install_scripts/files/rivet_fetch_tls.sh

```diff
@@ -0,0 +1,67 @@
+# Write script
+cat << 'EOF' > /usr/bin/rivet_fetch_tls.sh
+#!/usr/bin/env bash
+set -eu
+
+CERT_ID="job"
+STUB="/etc/__NAME__/tls/$CERT_ID"
+
+# Retry script every 5 seconds
+while true; do
+  response=$(
+    curl -f \
+      -H "Authorization: Bearer __SERVER_TOKEN__" \
+      "https://__DOMAIN_MAIN_API__/provision/datacenters/___DATACENTER_ID___/tls"
+  ) && break || sleep 5
+done
+
+echo "TLS received"
+
+# Write tls certs
+echo $response | jq -r .job_cert_pem > "${STUB}_cert.pem"
```

Comment:
What happens if jq errors? Will it write an empty cert? iirc we need `-o pipefail` right?